tests/e2e/workloads/bdi/bdi_base_class.py

```diff
@@ -0,0 +1,232 @@
+import pytest
+from datetime import datetime
+import logging
+import tempfile
+from ocs_ci.framework.testlib import E2ETest, workloads
+from ocs_ci.ocs import ocp
+from ocs_ci.utility import templating
+
+from ocs_ci.helpers.bdi_helpers import (
+    init_configure_workload_yaml,
+    init_data_load_yaml,
+    init_run_workload_yaml,
+    install_db2u,
+    clean_temp_files,
+)
+
+log = logging.getLogger(__name__)
+
+
+@workloads
+class TestBdiWorkloadBaseClass(E2ETest):
+    chart_dir = "/stable/ibm-db2warehouse/ibm_cloud_pak/pak_extensions"
+    chart_git_url = "https://github.com/IBM/charts"
+    tiller_namespace = "tiller"
+    helm_tar_file = "helm-v2.9.0-linux-amd64.tar.gz"
+    helm_url = "https://storage.googleapis.com/kubernetes-helm/" + helm_tar_file
+    helm_dir = None
+    helm_version = "v2.9.0"
+    tiller_template_url = "https://github.com/openshift/origin/raw/master/examples/helm/tiller-template.yaml"
+    bdi_dir = "/tmp/bdi_temp_dir"
+    db2u_project = "db2u-project"
+    ldap_r_n = "test-bdi-db-release-name"
+    ldap_r_p = "test-bdi-bluadmin-release-password"
+    db2u_r_n = ldap_r_n
+    db2u_r_p = "test-bdi-db-release-password"
+    ibm_cloud_key = "bIniFfn9hDJ4MlJZE2u_s0CpglC1hKgWs0W7IaLcdg7p"
```

Comment:
not a good idea to expose password / key in the code.
pleas change the way of using the password

Reply:
It's still in progress and will not be hard-coded here.